graphql_server/server_test.go

```diff
@@ -14,14 +15,27 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+type testSqlDriver struct {
```

Comment:
We can actually ask go to generate mocks for a given interface! We may also want to move this into a proper "LocalSqlDriver" or something since it is so easy to mock 🤷‍♀️